platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/AttributeMatchingArtifactVariantSelector.java

```diff
@@ -149,4 +165,11 @@ private static List<TransformedVariant> tryDisambiguate(
 
         return differentTransforms;
     }
+
+    private void throwAndMaybeReportProblem(AbstractResolutionFailureException e, boolean reportFailuresAsProblems) {
```

Comment:
Looks like this method isnt called and we can get rid of a bunch of wiring without it